### PR TITLE
Add SpaceHey

### PR DESCRIPTION
**Pull request by AnTheMaker**
Hi!
This PR adds https://spacehey.com as a site.
Let me know if I did it correctly!
Thx!
~ An

### CODE REVIEW

**Comment by AnTheMaker** [comment]
Oops, I messed up the file extension! Sorry!
Should be fixed now!

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by AnTheMaker** [comment]
Sure! There's no Documentation site for 2FA on SpaceHey yet.
Here's a screenshot of the 2FA account settings:
image

File: img/s/spacehey.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6-2.988 0-4.907 2.065-1.919 2.066-2.329 6.109Zm34.805 19.834.732-1.787-9.844-24.786q-.908-2.138-.908-3.105 0-1.026.527-1.904.528-.879 1.451-1.407.922-.527 1.919-.527 1.728 0 2.607 1.099.879 1.098 1.553 3.149l6.767 19.688 6.416-18.311q.762-2.227 1.377-3.486.616-1.26 1.304-1.7.688-.439 1.948-.439.908 0 1.743.483.835.484 1.289 1.304.455.82.455 1.729-.118.556-.381 1.582-.264 1.025-.645 2.08l-10.43 27.304q-1.347 3.604-2.636 5.655-1.289 2.05-3.413 3.149-2.124 1.099-5.728 1.099-3.515 0-5.273-.762-1.758-.762-1.758-2.783 0-1.377.835-2.124.835-.747 2.475-.747.645 0 1.26.176.762.175 1.319.175 1.377 0 2.168-.41.791-.41 1.42-1.421.63-1.01 1.451-2.973Zm-19.161-15.293h-15.644q.029 2.724 1.099 4.804 1.069 2.08 2.841 3.135 1.773 1.055 3.911 1.055 1.436 0 2.623-.337 1.186-.337 2.299-1.055 1.114-.718 2.051-1.538.938-.82 2.432-2.226.615-.528 1.758-.528 1.23 0 1.992.674t.762 1.904q0 1.084-.85 2.534-.85 1.451-2.564 2.784-1.713 1.333-4.306 2.212-2.593.879-5.962.879-7.705 0-11.983-4.395-4.277-4.395-4.277-11.924 0-3.545 1.055-6.577 1.054-3.032 3.076-5.2 2.021-2.168 4.98-3.325 2.959-1.158 6.563-1.158 4.687 0 8.042 1.978 3.354 1.977 5.024 5.112t1.67 6.387q0 3.018-1.728 3.911-1.729.894-4.864.894Z"/></g></svg>
```
Comment (Carlgo11) [inline img/s/spacehey.com.svg]: The viewbox is kind of odd. Here's a version with the viewbox corrected:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 283.1 266.58"><path fill="#1d4ed8" fill-rule="evenodd" d="M159.1 160v106.58h124V160a62.03 62.03 0 0 0-62-60 62.03 62.03 0 0 0-62 60zm17-115c0-24.83 20.2-45 45-45s45 20.17 45 45-20.2 45-45 45-45-20.16-45-45zM65.3 59.58H151v57.68a65.34 65.34 0 0 1-65.4 65.34H65.3C29.2 182.6 0 155.02 0 121.08c0-33.95 29.2-61.5 65.3-61.5ZM37.9 97.32v11.86q1.5-1.76 3-2.78 1.4-1.03 3.2-1.54 1.8-.51 3.8-.51 3 0 5.4 1.29 2.3 1.29 3.7 3.75.8 1.43 1.1 3.2.3 1.78.3 4.1V132q0 2.4-1.1 3.64-1.1 1.23-2.9 1.23-3.9 0-3.9-4.87v-13.5q0-3.84-1.2-5.9-1.1-2.07-4.3-2.07-2.2 0-3.9 1.21-1.7 1.22-2.5 3.33-.7 1.79-.7 6.36V132q0 2.38-1.1 3.62-1 1.25-2.9 1.25-4 0-4-4.87V97.32q0-2.44 1.1-3.65 1-1.22 2.9-1.22t2.9 1.23q1.1 1.23 1.1 3.64zM73 118.03h14.5q-.3-4.1-2.2-6.14-1.9-2.03-5.1-2.03-2.9 0-4.9 2.06-1.9 2.07-2.3 6.11zm34.8 19.83.8-1.78-9.9-24.79q-.9-2.14-.9-3.1 0-1.03.5-1.9.5-.89 1.5-1.42.9-.52 1.9-.52 1.7 0 2.6 1.1.9 1.1 1.5 3.15l6.8 19.68 6.4-18.3q.8-2.23 1.4-3.5.6-1.25 1.3-1.7.7-.43 2-.43.9 0 1.7.48.8.48 1.3 1.3.5.82.5 1.73-.2.56-.4 1.59-.3 1.02-.7 2.08l-10.4 27.3q-1.4 3.6-2.6 5.65-1.3 2.05-3.5 3.15-2.1 1.1-5.7 1.1-3.5 0-5.3-.76-1.7-.76-1.7-2.78 0-1.38.8-2.13.9-.74 2.5-.74.6 0 1.3.17.7.18 1.3.18 1.3 0 2.1-.41t1.5-1.42q.6-1.01 1.4-2.98zm-19.2-15.29H73q0 2.72 1.1 4.8 1.1 2.08 2.8 3.14 1.8 1.05 3.9 1.05 1.5 0 2.7-.33 1.2-.34 2.3-1.06 1.1-.72 2-1.54 1-.82 2.5-2.22.6-.53 1.7-.53 1.2 0 2 .67t.8 1.9q0 1.1-.9 2.54-.8 1.45-2.5 2.79-1.8 1.33-4.3 2.2-2.6.89-6 .89-7.7 0-12-4.4t-4.3-11.92q0-3.55 1.1-6.58 1-3.03 3.1-5.2 2-2.17 4.9-3.32 3-1.16 6.6-1.16 4.7 0 8 1.98 3.4 1.97 5.1 5.1 1.6 3.13 1.6 6.4 0 3.01-1.7 3.9-1.7.9-4.9.9z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thanks!

Since you seem to be the sole developer for SpaceHey, and a good one at that from the looks of it, you may want to look into supporting WebAuthn as well since that would be a step up from TOTP :wink:

**Comment by AnTheMaker** [comment]
Thanks for merging! :)

I totally will!
I was actually looking into buying Yubikeys today when I stumbled across 2fa.directory - I used it to see if the services I use would support Yubikeys... so as soon as I'll get the Yubikeys in the mail I can start to to implement WebAuthn on SpaceHey ;)

**Comment by Carlgo11** [comment]
Cloudflare are currently offering a [discount on Yubikeys](https://www.cloudflare.com/products/zero-trust/phishing-resistant-mfa/) for all their customers so I'd recommend you use that discount if you haven't bought your Yubikey yet.

**Pull request merged**